<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lazy DSi File Downloader</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>360</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="Home">
    <attribute name="title">
     <string>Home</string>
    </attribute>
    <widget class="QPushButton" name="next_page0">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>290</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>301</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/lazy-icon/assets/files/photos/lazy_300x300.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="HomeLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>331</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Lazy DSi File Downloader</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DiscordButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>171</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>DS⁽ⁱ⁾ Mode Hacking Discord </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="memory_pit">
    <attribute name="title">
     <string>Memory Pit</string>
    </attribute>
    <widget class="QPushButton" name="next_page1">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>290</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="previous_page0">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="MemoryPitCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Download Memory Pit?</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="fwComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>1.0 - 1.3 (USA, EUR, AUS, JPN)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.4 - 1.4.5 (USA, EUR, AUS, JPN)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>All versions (KOR, CHN)</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="memorypitlabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>105</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Select DSi Firmware + Region : </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="homebrew">
    <attribute name="title">
     <string>Homebrew</string>
    </attribute>
    <widget class="QPushButton" name="next_page2">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>290</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="previous_page1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="TWLMenuCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Download / Update TWiLight Menu ++?</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="dumpToolCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Download / Update dumpTool?</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="UnlaunchCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Download Unlaunch?</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>501</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="GodMode9iCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Download / Update GodMode9i?</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="hiyaCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>471</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Update hiyaCFW? (hiya MUST be already installed)</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ExtraHomebrewTab">
    <attribute name="title">
     <string>Extras</string>
    </attribute>
    <widget class="QPushButton" name="next_page4">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>290</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
    <widget class="QPushButton" name="previous_page4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>471</width>
       <height>231</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>469</width>
        <height>259</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
    </widget>
    <widget class="QLabel" name="MemoryPitInstruction_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>321</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Select Additional Homebrew for download :</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sd_card">
    <attribute name="title">
     <string>SD Card</string>
    </attribute>
    <widget class="QPushButton" name="StartButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>290</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="previous_page2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
    <widget class="QLabel" name="MemoryPitInstruction">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>Select your SD card root directory :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="SDDirectoryInput">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>341</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="FileDialogOpenButton">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Folder</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="download">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Download</string>
    </attribute>
    <widget class="QPushButton" name="Finish">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>290</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Finish</string>
     </property>
    </widget>
    <widget class="QPushButton" name="previous_page3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Previous</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="current_operation">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>411</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Current Operation</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="step_counter">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Step of</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
